### DISCUSSION TITLE
How to increase the size of svg built

### DISCUSSION THREAD

**Discussion by sagarr70** [adrianschlatter/ppf.datamatrix#9] [Q&A]
I want to create various SVG's of data matrix corresponding to some integer values .  But the svg resolution and size is very small any idea how can i increase that?

**Comment by adrianschlatter** [comment]
As svg is a vector graphics format, its "resolution" is infinite. Could you try to explain what exactly you mean by "resolution" and "size"?

**Comment by yatesdr** [comment]
I think he means the default viewport size or scaling.   Opening it up in any image software its size is the pixels width of the datamatrix itself.   I'm looking for the same answer at the moment for an automated workflow where I need the SVG to come out at an exact width for laser marking - say an 18x18 datamatrix needs to be displayed as 180px square.

Trying to use the SVG with software such as Lightburn is non-functional as it has single pixels representing the matrix since the matrix is generated as an SVG path, and that particular software requires a fill area to scale properly so it just shows up as lines.   To be fair the SVG generated isn't incorrect, it's just not that useful for some applications that require a filled shape instead of a pixel element.

I confirmed it does scale properly in Affinity and html displays though, so it obviously works for most common use cases.

---
Update:  Submitted PR that addresses output size scaling (#24)
